.bazelrc

```diff
@@ -17,3 +17,7 @@ build --deleted_packages tests/ios/frameworks/sources-with-prebuilt-binaries
 # when swiftmodule caching is enabled.
 # build --features=swift.cacheable_swiftmodules
 # build --features=swift.use_global_module_cache
+
+# Match this with the flag used in build-wrapper.sh
+build --compilation_mode=dbg
```

Comment:
why is this necessary? builds in this rules repo should work in all 3 compilation modes

Reply:
hmmm so the `bazel-out` directory name is a composition of compilation mode and configruation hash and other architecture parameters, right? If we don't specify this one, the framework search path will be `applebin.....-fastbuild-<somehash>` instead of `applebin.....-debug-<somehash>`. Am I correct here? (the change inside fixture also confirms this)

Reply:
so unless we make the build-wrapper respect the compilation mode in first place instead of hardcoding?

Reply:
either the build wrapper should respect the compilation mode or, my hunch is, we'll need a split transition so we can get the full matrix of settings for {debug,release}/{simulator,device}

Reply:
will do the `wrapper respect setting` one and do the split transition to get full matrix of settings for  {debug,release}/{simulator,device} in future

Reply:
okay turns out adding the macro in the first place is not needed because if we specify compile mode to be `dbg` under `.bzelrc`, it will be respected inside the build-warpper. unless i missed some case @amb

Reply:
In its current form, this PR has removed `compilation_mode=dbg` from the build-wrapper. Could you put it back? You can add it into the new `BAZEL_RULES_IOS_OPTIONS` build setting you have created and are using.

Reply:
we have discussed and agree that above comment is invalid as now the single point of truth is on user's .bazelrc file @amberdixon